src/sentry_plugins/asana/plugin.py

```diff
@@ -143,7 +143,7 @@ def get_client(self, user):
     def error_message_from_json(self, data):
         errors = data.get("errors")
         if errors:
-            return " ".join([e["message"] for e in errors])
+            return " ".join(e["message"] for e in errors)
```

Comment:
Oh I like this

Reply:
It does this automatically now with black or something?

Reply:
Oh this is pyupgrade, nice

Reply:
Yeah, it's good.